source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -192,16 +193,22 @@ def _getWordOffsetsInThisLine(self, offset, lineInfo):
 		# not more than two alphanumeric chars in a row.
 		# Inject two alphanumeric characters at the end to fix this.
 		lineText += "xx"
```

Comment:
Is this still necessary?

Reply:
I do think so, yes. Probably hard to verify.